.codeclimate.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+plugins:
+  eslint:
+    enabled: true
+    channel: "eslint-5"
+    config:
+      config: .eslintrc.json
+      extensions:
+        - .js
+        - .jsx
```

Comment:
You are missing a new line at the end of the file. 
See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Done. I have added the new line.